webview/src/experiments/components/table/TableHead.tsx

```diff
@@ -97,8 +97,6 @@ export const TableHead = ({
 
   const selectedForPlotsCount = getSelectedForPlotsCount(rows)
 
-  const firstExpColumnCellId = headerGroups[0].headers[0].id
```

Comment:
[F] This caused me much pain. The only way that the bug showed itself was in two stories. Seems more sensible to use a util function instead of passing this variable down through all of the components.

Reply:
Bug was that the experiment column shadow was shown when it didn't need to be.